### ISSUE TITLE
Transaction Pending UX/UI

### ISSUE THREAD

**Issue by adriennetilley** [HausDAO/daohaus-monorepo#17]

**Comment by adriennetilley** [comment]
![HAUS-loader](https://user-images.githubusercontent.com/42706432/160155717-3cfd9ff4-68f0-4fbe-9335-946f71b6abf6.svg)

Loader icon

**Comment by adriennetilley** [comment]
Yes! this is ready to go, sorry I missed this comment. This will be used everywhere there is transactions pending. 
To be discussed if we want to do the transaction history modal or wait on that till later.

**Comment by earth2travis** [comment]
Related to: https://github.com/HausDAO/daohaus-monorepo/issues/273
